devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -575,10 +578,17 @@ private def parseCommandResponse(String description) {
 				break
 			case 2:
 				responseMap.value = "unlocked"
+				if (responseMap.data) {
```

Comment:
Why not just one line here?

`responseMap.data = [notify : true]`

Is there a subtlety I'm missing?

Reply:
It's possible that responseMap.data already has some values in it by the time it reaches this line. For e.g. at line number 560 the responseMap.data can get assigned with a map and if that happens then doing `responseMap.data = [notify : true]` will overwrite the existing map with new one.